modules/input_suffix_check.nf

```diff
@@ -34,6 +34,18 @@ process input_suffix_check {
         """
 }
 
+process get_Citation {
+        label 'emboss'
+        publishDir "${params.output}/Docs", mode: 'copy'
+    output:
+        path("Citations.txt")
+    script:
+    """
+    git clone https://github.com/replikation/What_the_Phage.git
```

Comment:
* no
* the git is present in the '$workflow.projectDir'  variable please use a better approach
* you ca do it directly with '.fromPath (workflow.projectDir + "docs/file")' 
* and then just put a publish dir there (also without a process)